pkg/fission-cli/util/podutils.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 The Fission Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	v1 "k8s.io/api/core/v1"
+)
+
+// PodReadyStatus returns the number of ready containers and total containers present in pod
+func PodContainerReadyStatus(pod *v1.Pod) (readyContainers, noOfContainers int) {
```

Comment:
We already have a function `IsReadyPod` in pkg/utils/utils.go 
Please see it it can be used.

Reply:
`PodContainerReadyStatus` function returns the number of containers which are ready inside the pod and the total number of containers in the pod, which is used to populate the READY column like 2/2. Existing `IsReadyPod` return boolean value as whether the pod is ready or not.

Reply:
Got it. Could you please organize both functions in the same file? New one, which you have introduced. `podutils.go`

Reply:
Eventually, we plan to move all content to pkg/utils. We want to get rid of any other utils folders.

1. Could you please move podutils.go to `pkg/utils/podutils.go` ? 
2. Also, exiting `IsReadyPod` function needs to be moved from `pkg/utils/utils.go` to `pkg/utils/podutils.go`